shipyard-controller/handler/projecthandler.go

```diff
@@ -280,6 +293,21 @@ func (ph *ProjectHandler) CreateProject(c *gin.Context) {
 		SetBadRequestErrorResponse(c, fmt.Sprintf(InvalidRequestFormatMsg, err.Error()))
 		return
 	}
+
+	automaticProvisioningURL := os.Getenv("AUTOMATIC_PROVISIONING_URL")
+	if automaticProvisioningURL != "" && params.GitRemoteURL == "" {
+		provisioningData, err := getProvisioningData(*params.Name, automaticProvisioningURL)
```

Comment:
`getProvisioningData` is a weird name for a function which is provisioning a repository via an external API

Reply:
will be renamed in `RepositoryProvisioner`